lib/wallet/src/Cardano/Wallet/Read/Primitive/Coin.hs

```diff
@@ -0,0 +1,13 @@
+module Cardano.Wallet.Read.Primitive.Coin
+    ( unsafeFromLedger
+    ) where
+
+import GHC.Stack
+    ( HasCallStack )
+
+import qualified Cardano.Ledger.Coin as Ledger
+import qualified Cardano.Wallet.Primitive.Types.Coin as Coin
+import qualified Cardano.Wallet.Primitive.Types.Coin as Wallet
+
+unsafeFromLedger :: HasCallStack => Ledger.Coin -> Wallet.Coin
+unsafeFromLedger (Ledger.Coin c) = Coin.unsafeFromIntegral c
```

Comment:
This already exists: https://github.com/cardano-foundation/cardano-wallet/blob/master/lib/wallet/src/Cardano/Wallet/Shelley/Compatibility/Ledger.hs#L145-L153

Reply:
👌🏼 I'll fix it.